app/services/hulu_mapping_service.rb

```diff
@@ -0,0 +1,75 @@
+# Service to sync Hulu API with Kitsu
+class HuluMappingService
+  HULU_URL = 'https://distribution.hulu.com/api/v1/'.freeze
+
+  attr_reader :since
+
+  # Initialize service with optional datetime
+  def initialize(since = nil)
+    @since = since
+  end
+
+  # Grabs all available series from hulu, maps them as wel as their episodes
+  def sync_series_and_episodes
+    ActiveRecord::Base.logger = Logger.new(nil)
+    Chewy.strategy(:bypass)
+    offset = 0
+    limit = 10_000
+    series_res = Typhoeus.get(
+      "#{HULU_URL}/series?guid=#{guid}&primary_category=animation&offset=#{offset}&limit=#{limit}"
+    )
```

Comment:
Might I suggest splitting retrieval/deserialization from actually processing?

You could just make a method which yields the JSON of each series, I think that would make this code a bit easier to read